File: bitirme_2/jrclass/src/app/student-profile/Note.ts
import { Injectable } from "@angular/core";
import { Lesson } from "../ogrenci-giris/lesson";


@Injectable({
    providedIn: 'root'
  })
export class Note{
    noteID!:number;
    note1!:string;
    note2!:string;
    note3!:string;
    note4!:string;
    Lesson!:Lesson;
    lessonID!:number;   
}

File: BitirmeAPI/BitirmeAPI/Model/Note.cs
using System.Text.Json.Serialization;

namespace BitirmeAPI.Model
{
    public class Note
    {
        public int noteID { get; set; }
       public string? note1 { get; set; }
        public string? note2 { get; set; }
        public string? note3 { get; set; }
        public string? note4 { get; set; }

        [JsonIgnore]
        public Lesson Lesson { get; set; }
        public int lessonID { get; set; }
        

    }
}
